osbs/api.py

```diff
@@ -375,6 +375,12 @@ def _update_build_config_when_exist(self, build_json):
         old_nodeselector = existing_bc['spec'].pop('nodeSelector', None)
         logger.debug("removing build config's nodeSelector %s", old_nodeselector)
 
+        # Remove koji_task_id
+        koji_task_id = utils.graceful_chain_get(existing_bc, 'metadata', 'labels',
+                                                'koji-task-id')
+        logger.debug("removing koji-task-id %r", koji_task_id)
+        utils.graceful_chain_del(existing_bc, 'metadata', 'labels', 'koji-task-id')
```

Comment:
Wonder whether to guard these two lines with `if koji_task_id is not None:`, to avoid logs being too chatty.

Reply:
I think in most cases, this will not be None though.
Only when the previous build was submitted via `osbs-client` directly.